src/main/webapp/components/Navigation.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+/**
+ * Main List Element to be displayed on the Navigation.
+ *
+ * @param {href: a URL String to a resource, innerText: The display text}
+ */
+const NavigationItem = props =>
+    <li><a href = {props.href}>{props.innerText}</a></li>
+/**
+ * Main Navigation List.
+ * Instance will be embedded in ./Header component.
+ *
+ * @param {options: [option]} props arbituary object containing an array of
```

Comment:
typo: arbitrary; however I'd just drop this as it's a bit redundant.
If you have a `@typedef` for an `Option` type, you can refer to it here by `{@link Option}`.

Reply:
Updated in relation to [Documenting properties of values in an array](https://jsdoc.app/tags-param.html#parameters-with-properties)